Replicator/Replicator.cc

```diff
@@ -830,8 +830,14 @@ namespace litecore { namespace repl {
             MessageIn *response = progress.reply;
 
             if (response->isError()) {
-                logError("Error response from remote for request 'getCollections'");
-                return gotError(response);
+                gotError(response);
+                if (auto err = response->getError(); err.domain == "BLIP"_sl && err.code == 404) {
+                    constexpr slice sg3_0_specific = "No handler for BLIP request"_sl;
+                    if (err.message == sg3_0_specific) {
+                        logError("This Sync Gateway does not support named collections. Try configuring your replicator using the default collection");
```

Comment:
I suppose this is not specific to sync gateway though.  It could happen in P2P as well right?

Reply:
Right. For P2P, it would mean 3.1 Lite to 3.0 Lite. Should we cover this?
This Sync Gateway or your peer Lite"

Reply:
Sync Gateway or Remote Peer ?

Reply:
Slightly weird if I'm a user and I don't use default collection but the message tells me to configure the replicator with default collection. (But I guess it's understandable?)